frontend/js/src/recent/components/RecentDonors.tsx

```diff
@@ -0,0 +1,83 @@
+import { faThumbtack } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import * as React from "react";
+import { Link } from "react-router-dom";
+import {
+  getRecordingMBID,
+  getTrackName,
+  pinnedRecordingToListen,
+} from "../../utils/utils";
+
+type RecentDonorsCardProps = {
+  donors: DonationInfoWithPinnedRecording[];
+};
+
+function RecentDonorsCard(props: RecentDonorsCardProps) {
+  const { donors } = props;
+
+  if (!donors || donors.length === 0) {
+    return null;
+  }
+
+  return (
+    <>
+      <h3 className="text-center" style={{ marginTop: "10px" }}>
+        Recent Donors
+      </h3>
```

Comment:
We'll want a link to the donors page. Maybe inside the h3 here, with a `<small>` element?

![image](https://github.com/user-attachments/assets/ec997960-6fbb-42ba-a8ad-a17aeab5534c)